.circleci/config.yml

```diff
@@ -1,5 +1,181 @@
+              label: Setting up dependences
+              command: flutter pub add amplify_flutter && flutter pub add amplify_datastore && flutter pub add amplify_storage_s3 && flutter pub add amplify_analytics_pinpoint && flutter pub add amplify_auth_cognito
+              no_output_timeout: 20m
+          - run:
+              name: Adding integration_test package
+              command: 'sed -i -e "s/dev_dependencies:/dev_dependencies:\n  integration_test:\n    sdk: flutter/" ./pubspec.yaml && cat ./pubspec.yaml'
+          - run:
+              name: Update outdated dependences
+              command: flutter pub upgrade --major-versions
+          - run:
+              name: Adding amplifyconfig file
+              command: mv canaries/dummy_amplifyconfiguration.dart canaries/amplifyconfiguration.dart && cp canaries/amplifyconfiguration.dart amplified_todo/lib
+              working_directory: ~/flutter-canaries/
+          - run:
+              name: Adding test code
+              command: cp canaries/main.dart amplified_todo/lib && cp -r canaries/integration_test amplified_todo/integration_test && cp -r canaries/models amplified_todo/lib/models
+              working_directory: ~/flutter-canaries/
+
+
+    jobs:
+      flutter-android:
+        parameters:
+          flutter-version:
+            type: string
+        executor:
+          name: android/android-machine
+          resource-class: large
+          tag: 2021.10.1
+        working_directory: ~/flutter-canaries/amplified_todo
+        steps:
+          - checkout:
+              path: ~/flutter-canaries
+          - install-flutter:
+              flutter_branch: << parameters.flutter-version >>
+          - run:
+              name: Setting up project
+              command: cd ../ && flutter create amplified_todo
+          - run:
+              name: Update Android version
+              command: sed -i -e "s/minSdkVersion .*/minSdkVersion 21/" ./android/app/build.gradle && cat ./android/app/build.gradle
+          - setup-amplify-flutter-project
+          - flutter-orb/install_android_gradle:
+              app-dir: ./
+          - android/create-avd:
+              avd-name: flutter
+              install: true
+              system-image: system-images;android-29;default;x86
+          - android/start-emulator:
+              avd-name: flutter
+              post-emulator-launch-assemble-command: ls -lrt
+              restore-gradle-cache-find-args: ./android -name 'build.gradle'
+          - run-with-retry:
+              label: Run Flutter Build
+              command: flutter build apk --debug
+              no_output_timeout: 20m
+          - run-with-retry:
+              label: Run Flutter Tests
+              command: flutter test integration_test
+              no_output_timeout: 1h
+              retry-count: 5
+
+      flutter-ios:
+        parameters:
+          flutter-version:
+            type: string
+        executor: mac-executor
```

Comment:
Similar to above, seems like this is using a different executor than the iOS unit tests use. I think it will be easier to maintain these if we use the same executors. If there is a reason to use this one over the other one we are using, maybe we change the unit tests to use this one.